### DISCUSSION TITLE
Hacktoberfest 2022

### DISCUSSION THREAD

**Discussion by ArttuOll** [react-hookz/web#950] [General]
Hello everyone!

I have been challenged to complete this year's Hacktoberfest, which starts tomorrow. I was wondering, does this library have any plans
in participating in Hacktoberfest?

I have been using this library at work and been enjoying it a lot, so I would gladly be spending my Hacktoberfest effort for the benefit of this repository.

As far as I can tell, participating does not really require any extra effort from the library maintainers except for adding the "hacktoberfest" topic to the repo and reviewing the resulting PRs. (And dealing with the possible spam pull requests, which have been a problem some years ago, but I hear that it's gotten a a lot better.)  More info [here](https://hacktoberfest.com/participation/#maintainers).

**Comment by xobotyi** [comment]
Yah, shure, why not!
